repository/src/main/java/com/epam/esm/repository/builder/QueryBuilder.java

```diff
@@ -0,0 +1,88 @@
+package com.epam.esm.repository.builder;
+
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.stereotype.Component;
+
+import java.util.Iterator;
+import java.util.List;
+
+import static com.epam.esm.repository.Parameters.*;
+
+@Component
+public class QueryBuilder {
+
+    private QueryBuilder() {
+    }
+
+    public void nameOfATag(List<String> findBy, MapSqlParameterSource parameters, String tagName) {
```

Comment:
I have written several times about the naming rules of methods, be careful

Reply:
Please read about Builder pattern: https://dzone.com/articles/design-patterns-the-builder-pattern

Reply:
fixed